src/System.Memory/tests/ReadOnlySpan/CopyTo.cs

```diff
@@ -123,12 +126,16 @@ public static void Overlapping2()
             Assert.Equal<int>(expected, a);
         }
 
+        private sealed class CopyToLargeSizeTestFileCleanup : FileCleanupTestBase
+        {
+            public string TestFilePath => GetTestFilePath(0, "ReadOnlySpanTests_CopyToLargeSizeTest");
```

Comment:
Do we need two separate test files for the test? One for Span and for ReadOnlySpan?
Similarly, can we have just one implementation of the CopyToLargeSizeTestFileCleanup class?

Reply:
The string is just salt for the name - the infrastructure rolls in a Guid to create a new file regardless.